community-templates/metric-template.md

```diff
@@ -1,40 +1,51 @@
 # Name of Metric
 
-Question: 
+**Question:** Add the question here
 
-Tags: [3 to 5 Search keywords - working group, related terms, and synonyms comma delineated]
+## Overview
 
-## Description
-A description of what the metric is and what it captures.
+Short: 200 words max
+An overview of the metric should include:
 
-## Objectives
-Provide several listed points that clearly describe why someone wants to measure this metric and what can be known with it. As examples: 
+*   Explicitly describe what you are measuring in the first sentence. Be specific about what the data points are.
+*   How does measuring this metric help inform health and sustainability?
+*   If applicable, how does this metric inform DEI-related issues?
 
-- A community manager is interested in this metric because...
-- An OSPO manager is interested in this metric because...
-- An event organizer is interested in this metric because...  
+## Want to Know More?
 
-Each metric has the potential to influence diversity, equity, and inclusion on an open source project. Please consider these factors when framing metrics objectives. 
+<span markdown="1"><details>
 
```

Comment:
Let's remove this line as I think the <span> and <summary> lines needs to be together